### ISSUE TITLE
Face detection and the number of training steps

### ISSUE THREAD

**Issue by chakri1804** [thtrieu/darkflow#820]
Hello everyone,

I've tried training the yolov2-tiny net on the VGGFace2 dataset to attain a network to detect human faces.
The dataset set contains roughly 1,38,000 images of faces belonging to 500 different people.
I've trained for 72000 steps (roughly 14 epochs) and loss had saturated way before (at 11000ish steps)

The network seems to detect faces from the existing dataset but whenever it encounters new faces, its not giving any output.

Can anyone help me out ? 

PS: 
1. I took care of the classes and filters section, don't mention about them
2. Training the model from scratch - not loading any pre-trained weights because i had some issues loading them
3. Tried overfitting the model first - worked beautifully
4. Something about estimated convergence loss was mentioned in issue #80 , I'm pretty new to deep-learning realm, so could someone guide me out as to how i evaluate it ?
5. It's a vague question but how should i choose the number of epochs/steps i need to run the model for ?

**Comment by fogonthedowns** [comment]
I have a similar problem, I can detect images I train that are part of the training set but not new images. Did you find a solution to this problem? See my details on issue [80](https://github.com/thtrieu/darkflow/issues/80)

**Comment by chakri1804** [comment]
I've ditched the idea of using darkflow for facial detection as there were better solutions for this problem. Nevertheless, i've observed that after ridiculously long number of steps, the net was able to draw boxes on a few faces. Maybe its just training for a long time maybe ?
